.circleci/config.yml

```diff
@@ -186,23 +186,23 @@ workflows:
   build_and_test:
```

Comment:
I'll clean up circleci once we merge this, to avoid failing builds on other PRs.